README.md

```diff
@@ -44,73 +43,59 @@ You can then create a `Computer` in AiiDA:
+FirecREST api version [Enter 0 to get this info from server] [0]: 0
+Maximum file size for direct transfer (MB) [Enter 0 to get this info from server] [0]: 0
 Report: Configuring computer firecrest-client for user chrisj_sewell@hotmail.com.
 Success: firecrest-client successfully configured for chrisj_sewell@hotmail.com
 ```
 
+You can always check your config with
 ```console
 $ verdi computer show firecrest-client
----------------------------  ------------------------------------
-Label                        firecrest-client
-PK                           3
-UUID                         48813c55-1b2b-4afc-a1a1-e0d33a5b6868
-Description                  My FirecREST client plugin
-Hostname                     unused
-Transport type               firecrest
-Scheduler type               firecrest
-Work directory               /scratch/{username}/aiida/
-Shebang                      #!/bin/bash
-Mpirun command               mpirun -np {tot_num_mpiprocs}
-Default #procs/machine       2
-Default memory (kB)/machine  100
-Prepend text
-Append text
----------------------------  ------------------------------------
 ```
 
 See also the [pyfirecrest CLI](https://github.com/eth-cscs/pyfirecrest), for directly interacting with a FirecREST server.
 
-See [tests/test_calculation.py](tests/test_calculation.py) for a working example of how to use the plugin, via the AiiDA API.
-
-### Current Issues
 
-Simple calculations are now running successfully [in the tests](tests/test_calculation.py), however, there are still some critical issues, before this could be production ready:
+After this, everything should function normally through AiiDA with no problems.
+See [tests/test_calculation.py](tests/test_calculation.py) for a working example of how to use the plugin, via the AiiDA API.
 
-1. Currently uploading via firecrest changes `_aiidasubmit.sh` to `aiidasubmit.sh` 😱 ([see #191](https://github.com/eth-cscs/firecrest/issues/191)), so `metadata.options.submit_script_filename` should be set to this.
+If you encounter any problems/bug, please don't hesitate to open an issue on this repository.
 
-2. Handling of large (>5Mb) file uploads/downloads needs to be improved
+### Current Issues
 
-3. Handling of the client secret, which should likely not be stored in the database
+Calculations are now running successfully, however, there are still issues regarding efficiency, Could be improved:
 
-4. Monitoring / management of API request rates could to be improved
+1. Monitoring / management of API request rates could to be improved. Currently this is left up to PyFirecREST.
+2. Each transfer request includes 2 seconds of `sleep` time, imposed by `pyfirecrest`. One can takes use of their `async` client, but with current design of `aiida-core`, the gain will be minimum. (see the [closing comment of issue#94 on pyfirecrest](https://github.com/eth-cscs/pyfirecrest/issues/94) and [PR#6079 on aiida-core ](https://github.com/aiidateam/aiida-core/pull/6079))
```

Comment:
Are there differences between PyFirecREST and `pyfirecrest`? I feel like not, then we should be consistent. Here it is a bit confusing because the writing style changes between two consecutive points